### PR TITLE
[Port Mgr] Handle internal DPM result list

### PR DESCRIPTION
**Pull request by songxiaoyan**
1. Currently, only the gRPC interface invoking result is processed.
2. If the puslar interface fails to be invoked,puslar will be repeatedly invoked until it is successfully invoked.
3. On present, only the PM module processes the result.

### CODE REVIEW

**Review by xieus — COMMENTED** [review]
@songxiaoyan looks good with one minor comment.

File: services/port_manager/src/main/java/com/futurewei/alcor/portmanager/entity/PortStatusEnum.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package com.futurewei.alcor.portmanager.entity;
+
+public enum PortStatusEnum {
```
[the remark this answers is not in the document]
Comment (songxiaoyan) [reply services/port_manager/src/main/java/com/futurewei/alcor/portmanager/entity/PortStatusEnum.java]: ok，I have been removed the PortStatusEnum class and use the StatusEnum class.

**Review by xieus — APPROVED** [review]
LGTM!

File: services/data_plane_manager/src/main/java/com/futurewei/alcor/dataplane/controller/DpmController.java [hunk 2]
```diff
@@ -45,7 +45,7 @@ public InternalDPMResultList createNetworkConfiguration(@RequestBody NetworkConf
         return dpmService.createNetworkConfiguration(networkConfiguration);
     }
 
-    @PutMapping({"/network-configuration", "v4/network-configuration"})
+    @PostMapping({"/network-configuration", "v4/network-configuration"})
     @DurationStatistics
     public InternalDPMResultList updateNetworkConfiguration(@RequestBody NetworkConfiguration networkConfiguration) throws Exception {
         checkNetworkConfiguration(networkConfiguration);
```
Comment (cj-chung) [inline services/data_plane_manager/src/main/java/com/futurewei/alcor/dataplane/controller/DpmController.java:51]: @songxiaoyan  and @xieus This change will break dpm. There are two PostMapping with the same url after you change line48 from Put to Post.

**Pull request merged**